plume-front/src/main.rs

```diff
@@ -1,16 +1,17 @@
+#![recursion_limit="128"]
```

Comment:
what's this used for?

Reply:
Macros can be recursive in Rust, but to make sure every build has an end, the compiler defines a macro recursion limit. This line just tells Rust to make it a bit higher, because the `js!` macro calls itself a lot of time.